src/diagnostics/LogPointsManager.ts

```diff
@@ -92,9 +93,12 @@ class DebugSessionManager {
 
         const result: ISetLogpointResponse = await this._debugSession.customRequest("setLogpoint", { scriptId, lineNumber, columnNumber, expression });
         if (result.error !== undefined && result.error !== null) {
-            vscode.window.showErrorMessage("Cannot set logpoint successfully. Please refer to DEBUG CONSOLE for details.");
+            vscode.window.showErrorMessage("Cannot set logpoint successfully. Please refer to DEBUG CONSOLE and https://aka.ms/logpoints for details.");
             appserviceUtil.getOutputChannel().show();
-            appserviceUtil.getOutputChannel().appendLine(`Cannot set logpoint successfully, the error is ${result.error}.`);
+            const errorMessage = result.error.message === '' ?
+                'Cannot set logpoint. Please refer to- https://aka.ms/logpoints for more details.' :
+                `Cannot set logpoint, the error is "${result.error.message}". Please refer to- https://aka.ms/logpoints for more details.`;
```

Comment:
Most of the texts are the same. Maybe you could make it so the if is only about ? "" : `the error is "${result.error.message}`

Reply:
I prefer not. Who knows whether it will become something else in the translation. Having a logic like that might invalidate translation in some other language, which does not follow the same lexical order as English.

Reply:
We do this kind of things pretty often in VS (having placeholders in the strings {0}) and it's not a problem for localization...

Reply:
It is not a placeholder for a data item, it's a part of the sentence. Because it includes the "the error is" part.